changes.go

```diff
@@ -71,6 +71,7 @@ func FromData(config ChangesConfig) ([]Change, error) {
 			return nil, errors.Trace(err)
 		}
 	}
+	resolver.handleOffers(addedApplications)
```

Comment:
What do we do about `kubernetes` here, I think it's correct, but we should be sure.

Reply:
The function assumes that the bundle is already validated (at least according to the comment above its declaration) so we should probably deal with this when we are doing validations, right?

Reply:
I believe so.